src/main/java/geekbrains/java2/ht2_1/Board.java

```diff
@@ -0,0 +1,57 @@
+package geekbrains.java2.ht2_1;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+class Board {
+
+    static int SIZE = getBoardSize();
```

Comment:
\[Checkstyle\] ERROR: Variable 'SIZE' must be private and have accessor methods\.